LDAP-primer.md

```diff
@@ -308,22 +327,26 @@ changed to account for both:
     (|(uid=john.doe)(mail=john.doe))
 
 The filter would match an entry which has either `uid` or `mail` with the value "john.doe".
-In this case, the `uid` term would match. If the user entered "john.doe@example.com", the
+In this case, only the `uid` term would match. If the user entered "john.doe@example.com", the
 resulting filter would be:
 
     (|(uid=john.doe@example.com)(mail=john.doe@example.com))
 
-This filter would match the same entry, but in this case the `mail` term would evaluate
+This filter would match the same entry, but in this case only the `mail` term would evaluate
 to true.
 
+The above examples all use __equality__ filters. LDAP also supports other kinds of filters, notably
+__present__, __substrings__, __greaterOrEqual__, __lessOrEqual__, __approximate__, and
+__extensible__. These can all be combined with __or__ (as above), __and__ or __not__.
+
 ## 3.3. Other LDAP operations
 
 LDAP provides the full set of operations for maintaining the DIT: adding, modifying and deleting
 entries, the analogues of INSERT, UPDATE and DELETE operations on a relational database. A notable
-difference is that LDAP changes don't operate in bulk, but on a single entry. The protocol guarantees
-that all modifications of an entry performed in a single operation will be atomic: either all of them
-are visible, or none. The guarantee doesn't extend to a series of modifications or other operations
-that change the DIT.
+difference is that LDAP changes usually don't operate in bulk, but on a single entry. The protocol
```

Comment:
I meant "operations" here, which (as you know) don't do bulk changes, so I'll reformulate.